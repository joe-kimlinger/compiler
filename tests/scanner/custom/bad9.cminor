"here is a bad char:"
''
