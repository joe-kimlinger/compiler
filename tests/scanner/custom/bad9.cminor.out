STRING_LITERAL here is a bad char:
